src/serialization/form/FormParseNode.cs

```diff
@@ -143,21 +144,21 @@ public IEnumerable<T> GetCollectionOfPrimitiveValues<T>()
         }
     }
     /// <inheritdoc/>
-    public DateTimeOffset? GetDateTimeOffsetValue() => DateTimeOffset.TryParse(DecodedValue, out var result) ? result : null;
+    public DateTimeOffset? GetDateTimeOffsetValue() => DateTimeOffset.TryParse(DecodedValue, CultureInfo.InvariantCulture, DateTimeStyles.None, out var result) ? result : null;
```

Comment:
shouldn't the style be roundtrip here? https://learn.microsoft.com/en-us/dotnet/api/system.globalization.datetimestyles?view=net-8.0

Reply:
yes, it makes sense